definitions/variable/emissions/carbon-management.yaml

```diff
@@ -0,0 +1,97 @@
+# Carbon Capture is grouped into two levels of aggregation based on the fate of the captured CO2.
+# Level-1 reports whether captured CO2 is stored, utilized, or leaked
+# Level-2 reports additional disaggregation for long-lived vs. short-lived utilization.
+
+- Carbon Capture:
+    description: Captured carbon dioxide (CO2) from energy use in supply and demand sectors (IPCC category 1A, 1B) and from industrial processes 
+        (IPCC categories 2A, B, C, E), including emissions from international bunkers, net of negative emissions from bioenergy with CCS (BECCS), 
+        direct air carbon capture and storage (DACCS), and other CO2 removal (exclusive of land-use change) in these sectors
+    unit: Mt CO2/yr
+
+- Carbon Capture|Geologic Storage:
+    description: Captured carbon dioxide (CO2) that is injected into subsurface geologic storage reservoirs (e.g., in situ mineralization).
+    unit: Mt CO2/yr
+    note: Captured CO2 that is reacted at the earth's surface (e.g., ex situ mineralization) should be reported under Carbon Capture|Other Storage
+
+- Carbon Capture|Leakage:
```

Comment:
Should this be reported as a positive or negative value? I guess it should be negative so that it is clear that thus should be subtracted from total carbon capture?

Reply:
Good point! Indeed, negative values would make sense

Reply:
Actually I think all positive values make more sense.  CO2 leakage is still captured but not stored or utilized.  Sum of CO2 capture + leakage + utilization should equal to total CO2 captured

Reply:
Jay, you are right!